src/features/commentsSlice.ts

```diff
@@ -0,0 +1,79 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { fetchComments, postComment, deleteComment } from '../utils/thunks';
+
+type Comments = {
+  comments: Comment[],
+  isLoading: boolean,
+  hasError: boolean,
+};
+
+const startState: Comments = {
+  comments: [],
+  isLoading: false,
+  hasError: false,
+};
+
+const postCommentsSlice = createSlice({
+  name: 'comments',
+  initialState: startState,
+  reducers: {
+    removeComment: (state, action) => {
```

Comment:
it's redundant, you handle it here:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/5acf0e32-f50e-4c3d-8883-ff572ad648d8)

remove this action